bin/dropboxignore.sh

```diff
@@ -425,26 +421,28 @@ ignore_files () {
     for dropboxignore_file in $DROPBOX_IGNORE_FILES; do
       file_total_results=0
       # shellcheck disable=SC2013
-      if grep -q -P '^\s*!' "$dropboxignore_file" ; then
+      if grep -q -P '^\s*!' "$dropboxignore_file"; then
         echo -e "$YELLOW$(get_relative_path "$dropboxignore_file" "$BASE_FOLDER") contains exception patterns, will be ignored"
         continue
       fi
       while read -r file_pattern; do
-#      for file_pattern in $(grep -v -P '^\s*$|^\s*\#|^\s*!' "$dropboxignore_file"); do
+        #      for file_pattern in $(grep -v -P '^\s*$|^\s*\#|^\s*!' "$dropboxignore_file"); do
         file_pattern=${file_pattern%/}
         subdir="$(dirname "$file_pattern")"
         pattern="$(basename "$file_pattern")"
         n_results=0
         while read -r file_path; do
           ignore_file "$file_path"
-          (( n_results++ ))
+          ((n_results++))
         done < <(find "$(dirname "$dropboxignore_file")/$subdir" -name "$pattern")
-        file_total_results=$((file_total_results+n_results))
+        file_total_results=$((file_total_results + n_results))
       done < <(grep -v -P '^\s*$|^\s*\#|^\s*!' "$dropboxignore_file")
       log_debug "Matched files because of '$(get_relative_path "$dropboxignore_file" "$BASE_FOLDER")': $file_total_results"
     done
     echo -e "$BLUE
 Total number of ignored files: $TOTAL_N_IGNORED_FILES $DEFAULT"
+    echo -e "$BLUE
+Total number of ignored folders: $TOTAL_N_IGNORED_FOLDERS $DEFAULT"
```

Comment:
Let's use a single `echo` here:
```suggestion
    echo -e "$BLUE
Total number of ignored files: $TOTAL_N_IGNORED_FILES $DEFAULT
Total number of ignored folders: $TOTAL_N_IGNORED_FOLDERS $DEFAULT"
```